usecases/auth/authorization/types.go

```diff
@@ -346,25 +344,7 @@ func ShardsData(class string, shards ...string) []string {
 //
 //	A slice of strings representing the resource paths for the given class and tenants.
 func Tenants(class string, tenants ...string) []string {
-	class = schema.UppercaseClassesNames(class)[0]
-	if class == "" {
-		class = "*"
-	}
-
-	if len(tenants) == 0 || (len(tenants) == 1 && (tenants[0] == "" || tenants[0] == "*")) {
-		return []string{fmt.Sprintf("%s/collections/%s/tenants/*", TenantDomain, class)}
-	}
-
-	resources := make([]string, len(tenants))
-	for idx := range tenants {
-		if tenants[idx] == "" {
-			resources[idx] = fmt.Sprintf("%s/collections/%s/tenants/*", TenantDomain, class)
-		} else {
-			resources[idx] = fmt.Sprintf("%s/collections/%s/tenants/%s", TenantDomain, class, tenants[idx])
-		}
-	}
-
-	return resources
+	return ShardsMetadata(class, tenants...)
```

Comment:
I copied this on purpose for multiple reasons:
- when we do the RAFT migration we can remove `ShardsMetadata` as the `shard` part is not needed anymore. 
- tenant operations are supposed to work only on tenants, we can make that explicit by having tenant in the path name
- It makes it clearer in the caller path what you are trying to do, eg Tenant operations are calling `Tenants`

Reply:
> when we do the RAFT migration we can remove ShardsMetadata as the shard part is not needed anymore.

`shard` part  is needed for 
- backward compatible, otherwise there will be panics 
- having `/collection/` in general won't work best with current matchers because we have to be more specific, otherwise any future /collection/{whatever} would check when it shouldn't 
> tenant operations are supposed to work only on tenants, we can make that explicit by having tenant in the path name
that's already what is happening in this change, i don't want to expose the mix between tenant,shard to casbin, both at the end are shards , in case if we needed it in ST classes